pkg/volume/util/fsquota/project.go

```diff
@@ -301,18 +301,17 @@ func writeProjectFiles(fProjects *os.File, fProjid *os.File, writeProjid bool, l
 				}
 			}
 		}
+
+		err = os.Rename(tmpProjects, fProjects.Name())
```

Comment:
I'm not sure reordering this is correct. In the case that L298 returns an error, we don't want to rename `tmpProjects`